src/components/NewClusterModal.js

```diff
@@ -175,7 +175,7 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
     const { currentCluster } = this.props
     if (currentCluster) {
       const { clusterImages, jupyterUserScriptUri } = await Ajax().Jupyter.cluster(currentCluster.googleProject, currentCluster.clusterName).details()
-      const { dockerImage } = _.find({ tool: 'Jupyter' }, clusterImages)
+      const { dockerImage } = _.find({ imageType: 'Jupyter' }, clusterImages)
```

Comment:
Also need to change `dockerImage` to `imageUrl`... either by doing
```
{ imageUrl: dockerImage }
```
or by `s/dockerImage/imageUrl/g` below (preferred, IMO).